team13phase1.tex

```diff
@@ -149,7 +195,21 @@
   The system shall provide WLAN access at each property.
   \begin{fr}
   \item
-    Connecting a device to a property's WLAN shall require that the device be authenticated by a owner, manager, guest, or staff member of the property.
+    Connecting a device to a property's WLAN shall require that the device be authenticated by an owner, manager, guest, or staff member of the property.
+  \end{fr}
+\item
+  The system shall be capable of sending notifications to user's electronic devices.
+  \begin{fr}
+  \item
+    The devices supported for notification receipt shall include (but not be limited to) the devices listed in \ref{fr:software:devices}.
+  \item
+    The property owner and managers shall be notified if the system detects that one of its devices needs servicing.
+  \item
+    A property's lease-holding guest shall be notified if the system detects that the property's fridge is low on food.
+  \item
+    A property's owner, managers, guests, and maintenance staff shall be notified if the property's smoke detector senses any smoke.
+  \item
+    If data collected in \ref{fr:external:weather} indicates ongoing or imminent severe weather at a property, the owner, managers, and guests of that property are notified of that sever weather.
```

Comment:
When describing FRs, we can really only describe the behavior of the system. It becomes problematic to say "The property manager will be notified" because the system doesn't control the property manager. The best we can do is say something like "The system shall send a notification/alert to the property manager's phone when [event] occurs"